boards/seeeduino_xiao/doc.txt

```diff
@@ -0,0 +1,38 @@
+/**
+@defgroup    boards_seeeduino_XIAO Seeeduino_XIAO
```

Comment:
```suggestion
@defgroup    boards_seeeduino_xiao Seeeduino XIAO
```

The other files reference this group with the name in lowcase and Doxygen seems to be case sensitive. Please squash this right in.